data/little_games.csv

```diff
@@ -0,0 +1,5 @@
+game_id,season,type,date_time,away_team_id,home_team_id,away_goals,home_goals,venue,venue_link
```

Comment:
For now this will work, but we'll need to make a more inclusive test data set once we get further along.